core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
Game only has its own screen if you set its screen. If you don't set its screen, it has no screen to delegate render () calls to.

The time measurements you posted are actually pretty similar, but you might not see that if you only took these measurements once for each configuration. Try it several times with each to see if there's any real difference (there won't be).